include/dem/pw_contact_force.h

```diff
@@ -65,7 +65,8 @@ class PWContactForce
       &           pw_pairs_in_contact,
     const double &dt,
     std::unordered_map<types::particle_index, Tensor<1, dim>> &momentum,
-    std::unordered_map<types::particle_index, Tensor<1, dim>> &force) = 0;
+    std::unordered_map<types::particle_index, Tensor<1, dim>> &force,
+    const DEMSolverParameters<dim> &dem_parameters) = 0;
```

Comment:
???

Reply:
I needed a bunch of parameters, so i regrouped them with dem_parameters.